sections/homepage/Features/Features.tsx

```diff
@@ -181,58 +188,175 @@ const StyledFlexDivRow = styled(FlexDivRow)`
 	width: 826px;
 	flex-wrap: wrap;
 	justify-content: center;
+
+	${media.lessThan('sm')`
+		flex-direction: column;
+		width: 305px;
+	`}
+`;
+
+const FeatureCard = styled(FlexDivRow)`
+	/* background: linear-gradient(180deg, rgba(40, 39, 39, 0.5) 0%, rgba(25, 24, 24, 0.5) 100%); */
+	/* box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25), inset 0px 1px 0px rgba(255, 255, 255, 0.1),
+	inset 0px 0px 20px rgba(255, 255, 255, 0.03); */
+	/* width: 403px; */
+	/* height: 135px; */
```

Comment:
Maybe we could delete this if we're not using them anymore.

Reply:
Good catch. Yashar re-wrote the css here so I didn't notice these comments. I will remove them.